python-packages/presto_viewmodels_drf/presto_viewmodels_drf/serializers.py

```diff
@@ -0,0 +1,46 @@
+class OptinFieldsSerializerMixin:
```

Comment:
I'm not sure on naming. Is it unnecessarily verbose? Perhaps we should just go with `OptInFieldsMixin`? The package name indicates it's for serializer

(either way should be `OptIn` rather than `Optin`)